client/pages/fiches-inventaire/[id]/index.vue

```diff
@@ -0,0 +1,190 @@
+<script setup lang="ts">
+import { VIcon } from "@gouvminint/vue-dsfr";
+
+definePageMeta({
+  layout: 'basic'
+})
+
+const ficheId = useRoute().params.id as string
+const ficheInventaire = (await (new FicheInventaireApi().get(ficheId))).value
+
+const etape = ficheInventaire.etape
+const objectif = ficheInventaire.objectif
+const thematique = `${ficheInventaire.thematique.picto} ${ficheInventaire.thematique.label}`
+const titre = ficheInventaire.titre
+const illustration = ficheInventaire.illustrationUrl
+const anneeLancement = ficheInventaire.anneeDeLancement
+const porteur = ficheInventaire.porteur
+const site = ficheInventaire.lienSite
+const debutConsultation = ficheInventaire.debut
+const finConsultation = ficheInventaire.fin
+const statut = ficheInventaire.statut
+const condition = ficheInventaire.conditionParticipation
+const modalite = ficheInventaire.modaliteParticipation
+const nombreDeCaracteresMinimumPourAfficherContenu = 20
+
+const tabListName = 'Liste d’onglet'
+const tabs = [
+  {id: "lancement", title: 'Lancement de la consultation', content: ficheInventaire.etapeLancementHtml},
+  {id: "analyse", title: 'Analyse des résultats', content: ficheInventaire.etapeAnalyseHtml},
+  {id: "suivi", title: 'Suivi de la consultation', content: ficheInventaire.etapeSuiviHtml},
+]
+const activeTab = ref(tabs.findIndex((xx) => xx.title.toLowerCase() === etape.toLowerCase()) ?? 0)
```

Comment:
Ça marche pas ça, si ?
```suggestion
const activeTabIndex = tabs.findIndex((xx) => xx.title.toLowerCase() === etape.toLowerCase())
const activeTab = ref(activeTabIndex !== -1 ? activeTabIndex : 0)
```
Ou bien, si tu veux garder le tout scopé, et pas avoir de `activeTabIndex` **et** `activeTab`, au choix : 
```suggestion
const activeTab = ref((() => {
  const activeTabIndex = tabs.findIndex((xx) => xx.title.toLowerCase() === etape.toLowerCase())
  return activeTabIndex !== -1 ? activeTabIndex : 0
})())
```
```suggestion
const activeTab = ref((function () {
  const activeTabIndex = tabs.findIndex((xx) => xx.title.toLowerCase() === etape.toLowerCase())
  return activeTabIndex !== -1 ? activeTabIndex : 0
})())
```
```suggestion
const activeTab = ref((function getActiveTab() {
  const activeTabIndex = tabs.findIndex((xx) => xx.title.toLowerCase() === etape.toLowerCase())
  return activeTabIndex !== -1 ? activeTabIndex : 0
})())
```